config/routes.rb

```diff
@@ -21,4 +16,8 @@
   resources :test_passages, only: %i[show update] do
     get :result, on: :member
   end
+
+  namespace :admin do
+    resource :tests
```

Comment:
Ты объявил _единичный_ ресурс. Так делают, например, для ресурса "профиль пользователя". Следовательно, нет экшна `index`, есть только экшн `show`, и ему не нужен id
Тесты не являются единичным ресурсом, поэтому
```suggestion
    resources :tests
```